src/seedu/addressbook/data/person/Address.java

```diff
@@ -47,13 +42,4 @@ public boolean equals(Object other) {
                 || (other instanceof Address // instanceof handles nulls
                 && this.value.equals(((Address) other).value)); // state check
     }
-
-    @Override
-    public int hashCode() {
```

Comment:
You should also override the `hashCode` method if you override the `equals` method (don't rely on the parent class maintaining your invariant).